web/package.json

```diff
@@ -70,6 +71,7 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "postcss": "^8.4.5",
     "prettier": "^2.7.1",
+    "prettier-plugin-organize-imports": "^3.2.1",
```

Comment:
This will auto remove unused imports for react

Reply:
let me try but I saw we where using prettier for this projected so used the same

Reply:
we also use prettier in Portal, use use prettier + lint. You can check the settings there